src/index.html

```diff
@@ -83,6 +83,26 @@ <h2>Message based debugging and unit testing, system testing and regression test
                 </div>
             </div>
         </section>
+        <section class="companies p-3">
+            <div class="container">
+
+                <h2> Building with the Frank!Framework: </h2> 
+
+                <div class="row row--gap-xl row--break-sm mb-3">
+                    <div class="icon col">
+                        <img class="logo1" src="assets/images/logo/Logo_FRANK_Blok_RGB.svg" alt=""/>
+                    </div>
+
+                    <div class="icon col">
+                        <img class="logo2" src="assets/images/logo/logo-conduction.svg" alt="Monitoring icon"/>
+                    </div>
+                    <div class="icon col">
+                        <p>Your company?</p> <!--Go center ?-->
+                        <img class="plus" src="assets/images/logo/icons8-plus-100.png" alt="Monitoring icon"/><!--Go center?-->
```

Comment:
- Alt should be "plus icon"
- Comments not needed
- try to use svg instead of png